src/Uno.UWP/UI/Input/Preview.Injection/InputInjector.cs

```diff
@@ -98,4 +99,16 @@ public void InjectMouseInput(IEnumerable<InjectedInputMouseInfo> input)
 			_target.InjectPointerUpdated(args);
 		}
 	}
+
+	[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "IS_UNIT_TESTS", "__WASM__", "__NETSTD_REFERENCE__", "__MACOS__")]
+	public void InjectMouseInput(IEnumerable<(InjectedInputMouseInfo, VirtualKeyModifiers)> input)
```

Comment:
As discussed : would be great to implement the [`InjectKeyboardInput`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.input.preview.injection.inputinjector.injectkeyboardinput?view=winrt-22621) instead, but as long as this method is `internal` it's fine for now.